docker-compose.yml

```diff
@@ -64,6 +64,73 @@ services:
                 '
         depends_on:
             - article-store-postgres
+    article-store-airflow-flower:
+        build:
+          context: ./workflow/airflow_libero
+        depends_on:
+            - article-store-airflow-redis
+        environment:
+            - EXECUTOR=Celery
+        ports:
+            - "5555:5555"
+        command: flower
+    article-store-airflow-postgres:
+        image: postgres:9.6.8-alpine
+        environment:
+            - POSTGRES_USER=airflow
+            - POSTGRES_PASSWORD=airflow
+            - POSTGRES_DB=airflow
+            - PGDATA=/var/lib/postgresql/data
+        volumes:
+            - 'article-store-airflow-postgres:/var/lib/postgresql/data'
+    article-store-airflow-redis:
+        image: 'redis:3.2.7'
+    article-store-airflow-scheduler:
+        build:
+          context: ./workflow/airflow_libero
+        depends_on:
+            - article-store-airflow-webserver
+        volumes:
+            - './workflow/airflow_libero/dags:/usr/local/airflow/dags'
```

Comment:
There's a few things in play here:

1. Containerised Airflow
2. Containerised Libero Airflow
3. Libero DAGs
4. Custom DAGs

I'm interested to see how this could look separated. (1 might be ignored depending on how 2 works.)

Reply:
You can probably ignore 1, really Its just containerised Libero Airflow and the need for x2 volumes (plugins and dags).

In the article-store, airflow integration PR I will be stripping out bits from the `libero_airflow` package and making it only contain the config bits so it should tidy it up.